DDL.sql

```diff
@@ -0,0 +1,27 @@
+create database if not exists bikepoint;
+use bikepoint;
+drop table if exists point;
+drop table if exists addons;
+CREATE TABLE IF NOT EXISTS `point` (
+	`id` VARCHAR(300) unique,
+	`url` VARCHAR(300) NULL,
+	`commonName` VARCHAR(300) NULL,
+	`placeType` VARCHAR(300) NULL,
+	`children` JSON NULL,
+	`childrenUrls` JSON NULL,	
+	`lat` FLOAT NULL,
+	`lon` FLOAT NULL,
+	`bikes` int NULL,
+	`spaces` int NULL,
+	`docks` int NULL,
+	  CONSTRAINT prkey1 PRIMARY KEY (id)
+);
+CREATE TABLE IF NOT EXISTS `addons` (
```

Comment:
Since you have already stored the data for bike availability in the 'point' table, this table might be redundant.